app/main.py

```diff
@@ -1,3 +1,73 @@
+from __future__ import annotations
+from typing import Any
+
+
+# solution
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: Any) -> None:
```

Comment:
Consider adding a type check in the `__init__` method to ensure `km` is a number (int or float) to prevent potential errors.